azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -469,8 +466,12 @@ private void loadQueuedFlows() throws ExecutorManagerException {
     if (runningCandidate != null) {
       executionIds.addAll(getRunningFlowsHelper(projectId, flowId, Lists.newArrayList(runningCandidate)));
     }
-    executionIds.addAll(getRunningFlowsHelper(projectId, flowId,
-      runningFlows.values()));
+    try {
```

Comment:
Will this confuse with getExecutableFlows()? In the future, queuedFlows will be totally removed, so getRunningFlows will only return runningFlows.